packages/date-time/src/components/CalendarDayGrid/CalendarDayGrid.base.tsx

```diff
@@ -615,19 +698,38 @@ export class CalendarDayGridBase extends BaseComponent<ICalendarDayGridProps, IC
 
   private onMouseOverDay = (day: IDayInfo) => {
     return (ev: React.MouseEvent<HTMLElement>) => {
-      const dayRefs = this.getRefsInRangeOfDay(day);
+      const dayInfos = this.getDayInfosInRangeOfDay(day);
+      const dayRefs = this.getRefsFromDayInfos(dayInfos);
 
-      dayRefs.forEach((dayRef: HTMLElement) => {
+      dayRefs.forEach((dayRef: HTMLElement, index: number) => {
         if (dayRef) {
           dayRef.classList.add('ms-CalendarDay-hoverStyle');
+          if (
+            !dayInfos[index].isSelected &&
+            this.props.dateRangeType === DateRangeType.Day &&
+            this.props.daysToSelectInDayView &&
+            this.props.daysToSelectInDayView > 1
+          ) {
+            // remove the static classes first to overwrite them
+            dayRef.classList.remove(this.classNames.bottomLeftCornerDate!);
```

Comment:
option: you can pass all the class names in a single remove() call:

Reply:
Wow I totally missed that. I saw that you couldn't pass them as a single string so I assumed you had to call individually but that makes more sense.